src/fde.h

```diff
@@ -84,8 +84,25 @@ class fde
     /// use I/O methods that maintain an internal-to-them buffer
     void useBufferedIo(READ_HANDLER *, WRITE_HANDLER *);
 
-    int read(int fd, char *buf, int len) { return readMethod_(fd, buf, len); }
-    int write(int fd, const char *buf, int len) { return writeMethod_(fd, buf, len); }
+    /// call the read method set by setIo()
```

Comment:
I accidentally dropped the terminating period in my earlier suggestion, probably while being confused by the missing `call` capitalization. Sorry. That period may become important for Doxygen "briefs" and improves overall multiline comment consistency/shape, so please fix this:

```suggestion
    /// Call the read method set by setIo(). 
```

While we are at it, besides setIo(), there are actually at least two more public functions that set I/O methods: useDefaultIo() and useBufferedIo(). Let's avoid the implication that only setIo() matters here. For example:

```suggestion
    /// Call the read method set by setIo() or similar. 
```

or just drop that non-essential detail completely:

```suggestion
    /// Call the previously set read I/O method.
```